released/packages/coq-ssreflect/coq-ssreflect.1.5.0/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "2.0"
+authors: [ "Jeremy Avigad <>" "Andrea Asperti <>" "Stephane Le Roux <>" "Yves Bertot <>" "Laurence Rideau <>" "Enrico Tassi <>" "Ioana Pasca <>" "Georges Gonthier <>" "Sidi Ould Biha <>" "Cyril Cohen <>" "Francois Garillot <>" "Alexey Solovyev <>" "Russell O'Connor <>" "Laurent Théry <>" "Assia Mahboubi <>" ]
+maintainer: "thomas.braibant@gmail.com"
+homepage: "http://ssr.msr-inria.inria.fr/"
+license: "CeCILL-B"
+build: [
+  [make "-j%{jobs}%"]
+  [make "install"]
```

Comment:
Like for another package, I'm worried this will not work with `bwrap` (with `make "install"` in the `build`). Can you please verify that installation is not a no-op?